lua/telescope/actions/init.lua

```diff
@@ -82,33 +82,33 @@ local append_to_history = function(prompt_bufnr)
 end
 
 --- Move the selection to the next entry
----@param prompt_bufnr number: The prompt bufnr
+---@param prompt_bufnr integer: The prompt bufnr
```

Comment:
Be aware that this may break stricter tooling (emmyluals) since the Neovim Lua API is not `integer` typed.

Reply:
Very well!